### DISCUSSION TITLE
@docker backup location & immutable snapshots

### DISCUSSION THREAD

**Discussion by takarangi** [007revad/Synology_app_mover#123] [Q&A]
Hello

First, thank you for your script! But i have a question and a suggestion.

I've defined my backup location in the config file as you explain. But when I run the script to move container manager I've said y to the backup option and waited indefinitely for @docker to backup (nearly in the next morning the message alerting for the waiting time was still there). I then pressed enter and some more lines were printed in the shell telling me the move was successful (which it was, container manager is in volume 2 and running). But there is nothing in the backup folder related to the backup! Previously i created a subolder there with the docker compose of my projects but I think it wouldn't impede the backup. Can you tell what maybe was the problem or where is the backup?

I then went to try to change the volume of the docker shared folder but it wasn't possible as I had activated immutable snapshots (only for 7 days). So my suggestion, although it is not related to the script, is to alert to that question in the guide, so maybe those interested can deactivate immutable snapshots sooner, before the migration. 

I also went and migrated synology photos, drive and tailscale with success!

**Comment by 007revad** [comment]
The backuppath setting in syno_app_mover.conf is only used when you select Backup or Restore. It's on the readme but I should add that to the config file.

When moving Container Manager or Docker it asks if you want to backup `@docker`. If you answer 'y' the script copies `@docker` to `@docker_backup` on the same volume that Container Manager or Docker is currently installed on. This is because if the move is incomplete due to an error the `@docker_backup` will be needed for me to be able to help them recover their containers.

**Reply by takarangi** [reply]
Thanks. I could find it using the shell (doesn't appear in file station). But now in volume1 I have @docker and @docker_backup and in volume2 @docker.

 Can i delete the ones in volume1? Or @docker in volume1 is still useful for some DSM processes?

**Reply by 007revad** [reply]
You can delete /volume1/@docker and /volume1/@docker_backup
